assets/i18n/en_US.json

```diff
@@ -152,10 +152,10 @@
     },
     "patchItem": {
         "unsupportedDialogText": "Selecting this patch may result in patching errors.\n\nApp version: {packageVersion}\nSupported versions:\n{supportedVersions}",
-        "unsupportedPatchVersion": "Patch is not supported for this app version. Enable the experimental toggle in settings to proceed.",
+        "unsupportedPatchVersion": "Patch is not supported for this app version.",
         "unsupportedRequiredOption": "This patch contains a required option that is not supported by this app",
 
-        "patchesChangeWarningDialogText": "It is recommended to use the default selection of patches because changing it may cause unexpected issues.\n\nIf you know what you are doing, you can enable \"Enable changing selection\" in the settings.",
+        "patchesChangeWarningDialogText": "It is recommended to use the default selection of patches because changing it may cause unexpected issues.\n\nIf you know what you are doing, you can enable \"Allow changing patch selections\" in the settings.",
```

Comment:
`It is recommended to use the patch selections and options. Changing it may result in unexpected issues.\n\nYou can disable this warning by turning on \"Allow changing patch selections\" in settings.`

Reply:
> `It is recommended to use the patch selections and options...`

I'll do `It is recommended to use the default patch selections and options...`

Reply:
Actually, we can use the starting sentence just fine. We need a better ending sentence. How about: `You'll need to turn on "Allow changing patch selection" in settings before changing any patch selection.`

Reply:
> Actually, we can use the starting sentence just fine. 

I don't understand how `It is recommended to use the patch selections and options.` makes sense. It doesn't say which patch selection. It's effectively meaningless.

Reply:
@PalmDevs

Reply:
> I don't understand how "`It is recommended to use the patch selections and options.`" makes sense. It doesn't say which patch selection. It's effectively meaningless.
The default, I meant.

> Also btw, I'm assuming I should keep the "Changing them may cause unexpected issues."
Yes.